Production/test/condorSub/dict_RunIIFall17MiniAODv2_diboson.py

```diff
@@ -0,0 +1,30 @@
+#Made in python using:
+'''
+import glob

```

Comment:
do we really need to have this as a block comment in every condor dict? Can't it just be a script with the patterns specified as an argument?

Reply:
It could be, but I haven't gotten around to writing a script. Do you definitely want that before you approve this?

Reply:
I would prefer to avoid this amount of code duplication persisting indefinitely. It seems like the first version of the script could be pretty simple.

Reply:
Okay, give me a moment.

Reply:
Okay, I have updated the PR with a new script and I have tested that it works.